hivelvet-docs/build-installer-dev.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Looks good. I think renaming it to `build.sh` or `build-docs.sh` is better. Using `build-""installer"".sh`, installer is the installer frontend app.

Reply:
okay, i will rename it